File: Solutions/_1.cs
public int[] TwoSum(int[] nums, int target)
{
    Hashtable hash = new Hashtable();
    List<int> result = new List<int>();
    for (int i = 0; i < nums.Count(); i++)
    {
        var b = target - nums[i];
        if (hash.ContainsKey(b))
        {
            result.Add(Array.IndexOf(nums, b));
            result.Add(i);
            return result.ToArray();
        }
        hash[nums[i]] = i;
    }
    return null;
}

File: Solutions/_1.go
package main

func twoSum(nums []int, target int) []int {
	var returnVal []int
	hashmap := make(map[int]int)
	for i, v := range nums {
		b := target - v
		if k, ok := hashmap[b]; ok {
			returnVal = append(returnVal, k, i)
			return returnVal
		}
		hashmap[v] = i
	}
	return returnVal
}
